.github/workflows/e2e.yaml

```
@@ -0,0 +1,39 @@
+name: e2e-mock-provider-tests
+on: 
+  workflow_dispatch:
+    inputs:
+      registry:
+        description: 'Registry'
+        required: true
+        default: 'gcr.io/k8s-staging-csi-secrets-store'
+      driverImage:
+        description: 'Secret Store CSI driver image'
+        required: true
+        default: 'driver'
+      driverImageVersion:
+        description: 'Secret Store CSI driver image version'
+        required: true
+        default: 'v0.3.0'
+jobs:
+  e2e-test:
```

```suggestion
  e2e_test:
```

nit: for consistency